docs/settings/general-infra-logs-ui-settings.asciidoc

```diff
@@ -21,4 +21,7 @@ Field used to identify hosts. Defaults to `host.name`.
 Field used to identify Docker containers. Defaults to `container.id`.
 
 `xpack.infra.sources.default.fields.pod`::
-Field used to identify Kubernetes pods. Defaults to `kubernetes.pod.uid`.
\ No newline at end of file
+Field used to identify Kubernetes pods. Defaults to `kubernetes.pod.uid`.
+
+`xpack.infra.alerting.inventory_threshold.group_by_page_size`::
+Controls the size of the composite aggregations used by the Inventory Threshold to retrieve all the hosts. Defaults to `10000`.
```

Comment:
Nit: I'd recommend to write the number using the thousand separator for better readability. IIUC YAML supports the underscore separator (`10_000`) if this comment should also be written to the default `config.yml`?